<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="RAJESH-PC" ignored="0" name="TestonFirefox" tests="4" failures="1" timestamp="19 Oct 2019 07:15:33 GMT" time="23.412" errors="0">
  <testcase name="TestCase1_verifyTitle" time="0.019" classname="com.vtiger.tests.loginTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.vtiger.tests.loginTest.TestCase1_verifyTitle(loginTest.java:37)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:748)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- TestCase1_verifyTitle -->
  <testcase name="TestCase2_verifyLogo" time="0.23" classname="com.vtiger.tests.loginTest"/>
  <testcase name="TestCase3_InvalidLogin" time="2.366" classname="com.vtiger.tests.loginTest"/>
  <testcase name="TestCase4_Login" time="9.082" classname="com.vtiger.tests.loginTest"/>
</testsuite> <!-- TestonFirefox -->
